mcc/odin/odin/snapshot_list_test.go

```diff
@@ -17,3 +21,39 @@ var exampleSnapshot1 = &rds.DBSnapshot{
 	MasterUsername:       aws.String("owner"),
 	Status:               aws.String("available"),
 }
+
+type listSnapshotsCase struct {
+	testCase
+	name       string
+	snapshots  []*rds.DBSnapshot
+	instanceID string
+}
+
+var listSnapshotsCases = []listSnapshotsCase{
+	// No snapshots for any instance
+	{
+		testCase: testCase{
+			expected:      []*rds.DBSnapshot{},
+			expectedError: "",
+		},
+		name:       "No snapshots for any instance",
+		snapshots:  []*rds.DBSnapshot{},
+		instanceID: "",
+	},
+}
+
+func TestListSnapshots(t *testing.T) {
+	svc := newMockRDSClient()
+	for _, test := range listSnapshotsCases {
```

Comment:
yes. Range iterates over all elements in the rightmost variable, one at a time. It assigns the key to the first returned element (which is dropped by the `_` keyword so it is ignored) and the value to the `test` variable. Every time the `for` loop runs, test holds a different value until the end of `listSnapshotsCases`.